core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -56,71 +56,7 @@
-               alt="Publish to Oppia Library">
-              <loading-dots ng-show="loadingDotsAreShown"></loading-dots>
-            </i>
-            <span class="oppia-save-publish-button-label">
-              Publishing
-              <loading-dots ng-show="loadingDotsAreShown"></loading-dots>
-            </span>
-          </span>
-        </button>
-      </div>
-    </li>
-
-    <li ng-if="isEditableOutsideTutorialMode()">
-      <div dropdown class="btn-group" style="margin-right: 10px; margin-top: 8px;"
-           title="<[getSaveButtonTooltip()]>">
-        <button id="tutorialSaveButton" class="btn btn-default oppia-save-draft-button protractor-test-save-changes" ng-class="{'btn-success': isExplorationSaveable(), 'oppia-save-draft-button-padding': getChangeListLength() && !isPrivate()}" ng-click="saveChanges()" ng-disabled="!isExplorationSaveable()">
-          <span ng-if="!saveIsInProcess">
-            <span ng-if="isPrivate()">
-              <i class="material-icons md-18 md-dark oppia-save-publish-button-icon"
-                 alt="Save Draft">&#xE161;</i>
-              <span class="oppia-save-publish-button-label">Save Draft</span>
-              <span class="hidden-sm hidden-xs" ng-if="getChangeListLength()" style="opacity: 0.5">(<[getChangeListLength()]>)</span>
-            </span>
-            <span ng-if="!isPrivate()" title="Publish Changes">
-              <i class="material-icons md-18 md-dark oppia-save-publish-button-icon"
-                 alt="Publish Changes">&#xE2C3;</i>
-              <span class="oppia-save-publish-button-label">Publish Changes</span>
-              <span class="hidden-sm hidden-xs" ng-if="getChangeListLength()" style="opacity: 0.5">(<[getChangeListLength()]>)</span>
-            </span>
-          </span>
-          <span ng-if="saveIsInProcess">
-            <span ng-if="isPrivate()" class="oppia-save-publish-button-label">
-              Saving
-            </span>
-            <span ng-if="!isPrivate()" class="oppia-save-publish-button-label">
-              Publishing
-            </span>
-            <loading-dots ng-show="loadingDotsAreShown"></loading-dots>
-          </span>
-        </button>
-        <button type="button" class="btn btn-default dropdown-toggle protractor-test-save-discard-toggle"
-                ng-disabled="!getChangeListLength()" dropdown-toggle>
-          <span class="caret"></span>
-        </button>
-        <ul class="dropdown-menu" role="menu" style="min-width: 120px; right: inherit;" ng-style="{ width: getChangeListLength() && !isPrivate() ? '150px' : '120px' }">
-          <li><a ng-click="discardChanges()" ng-class="{'oppia-disabled-link': !getChangeListLength()}" class="protractor-test-discard-changes" title="Discard all pending changes.">Discard Draft</a></li>
-        </ul>
-      </div>
-    </li>
-  </ul>
+<exploration-save-and-publish-buttons></exploration-save-and-publish-buttons>
```

Comment:
I think I missed this in the previous review, but could you indent this line and line 57 by 2 since they are both subsidiary to the 'block' tag?